src/contentScript/deepl.js

```diff
@@ -39,16 +39,16 @@ void (function () {
             `button[dl-test|=translator-lang-option-${targetLanguage}]`
           );
           buttonSelectedTargetLanguage.click();
-        } else if (target_textarea.value && text === source_textarea.value) {
-          resolve(target_textarea.value);
+        } else if (target_textarea.textContent && text === source_textarea.textContent) {
+          resolve(target_textarea.textContent);
           return;
         }
       } catch (e) {
         console.error(e);
       }
 
       // set the source language, the translation will start immediately
-      source_textarea.value = text;
+      source_textarea.firstChild.firstChild.textContent = text;
```

Comment:
only saying `source_textarea.textContent = text` does not make translation start after dispatchEvent

Reply:
This is because DeepL no longer uses the native textarea.